torch/utils/data/datapipes/dataframe/dataframes.py

```diff
@@ -416,7 +416,7 @@ def __getattr__(self, attrname):  # ?
 
 @functional_datapipe('trace_as_dataframe')
 class DataFrameTracer(CaptureDataFrameWithDataPipeOps, IterDataPipe):  # type: ignore[misc]
-    source_datapipe = None
+    source_datapipe: Optional[Any] = None
```

Comment:
equivalent to ```suggestion
    source_datapipe: Optional = None
```
right?